lib/wrap-guide-element.coffee

```diff
@@ -89,6 +90,9 @@ class WrapGuideElement extends HTMLDivElement
     else
       @style.display = 'none'
 
+  setTooltip: ->
+    @title = "This is the wrap guide."
```

Comment:
I was just thinking of tooltip text of "wrap-guide" and trusting people to search the command palette. But yeah, it doesn't cost us anything to go all the way :grinning: